### PR TITLE
Update index.md for Python

### PR DESCRIPTION
**Pull request by guangyuexu**
Clarify that Python 3 is required and Python 2.7 is not supported to run WebDriver tests with Selenium 4.

### CODE REVIEW

File: microsoft-edge/webdriver-chromium/index.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ To begin writing automated tests, make sure the Edge WebDriver version you insta
 <!-- ====================================================================== -->
 ## Choose a WebDriver testing framework
 
-After downloading Edge WebDriver, the last component you must download is a WebDriver testing framework.  Test authors use WebDriver testing frameworks to write end-to-end tests and automate browsers.  A WebDriver testing framework provides a language-specific interface that translates your code into commands that Edge WebDriver runs in Microsoft Edge.  WebDriver testing frameworks exist for all major platforms and languages, such as Python, Java, C#, Ruby, and JavaScript.
+After downloading Edge WebDriver, the last component you must download is a WebDriver testing framework.  Test authors use WebDriver testing frameworks to write end-to-end tests and automate browsers.  A WebDriver testing framework provides a language-specific interface that translates your code into commands that Edge WebDriver runs in Microsoft Edge.  WebDriver testing frameworks exist for all major platforms and languages, such as Python, Java, C#, Ruby, and JavaScript. Note that for Python users, Python 3 is required and Python 2.7 is not supported.
```
Comment (bwalderman) [inline microsoft-edge/webdriver-chromium/index.md:67]: Please move the note to the "Using Selenium 4" section. Python 3 is only required if you want to follow our Selenium 4 examples.

Technically, you can use a WebDriver client written in any language, including Python 2.7, and we don't place restrictions on what client you use and what language it's written in as long as it is W3C compliant.

**Review by bwalderman — CHANGES_REQUESTED** [review]

**Comment by captainbrosset** [comment]
Thanks for opening this. I've also added Jason as a reviewer. Once Brandon approves this, Jason can take a look and approve as well.

File: microsoft-edge/webdriver-chromium/index.md (hunk 1 above)
Comment (guangyuexu) [reply microsoft-edge/webdriver-chromium/index.md]: Moved notes to `Using Selenium 4` sesion.

**Review by bwalderman — APPROVED** [review]

File: microsoft-edge/webdriver-chromium/index.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ This article provides instructions for using the Selenium framework, but you can
 
 ### Using Selenium 4
 
-Selenium WebDriver is an open-source testing framework that can be used on any platform, and provides language bindings for Java, Python, C#, Ruby, and JavaScript.
+Selenium WebDriver is an open-source testing framework that can be used on any platform, and provides language bindings for Java, Python, C#, Ruby, and JavaScript. Note that for Python users, Python 3 is required to run Selenium 4 tests, Python 2.7 is not supported.
```
Comment (mikehoffms) [inline microsoft-edge/webdriver-chromium/index.md:74]: Python 3 is only required _if_ you want to follow our Selenium 4 examples.

You can use a WebDriver client written in any language, _including Python 2.7_. You can use clients and languages as long as they are W3C-compliant.

Comment (JasonAndrewWriter) [reply microsoft-edge/webdriver-chromium/index.md:74]: Note: Python 3 is required to run Selenium 4 tests.  (Python 2.7 is not supported.)

Comment (JasonAndrewWriter) [reply microsoft-edge/webdriver-chromium/index.md:74]: This is just a recommendation to streamline the language.

Comment (guangyuexu) [reply microsoft-edge/webdriver-chromium/index.md]: Thank you for the suggestions. I have updated the doc as suggested. Please merge the PR if this looks good. (I don't have permission to merge this PR to the repo.) @JasonAndrewWriter @mikehoffms

**Review by Reezaali — APPROVED** [review]

**Pull request merged**